daycare/conftest.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from .models import DayCare, Image
+
+EMAIL = "test@gmail.com"
+USERNAME = "testuser01"
+PASSWORD = "pass"
+NAME = "Puppies"
+DESCRIPTION = "This is the first daycare test"
+PRICE_PER_DAY = 10
+CAPACITY = 50
+AREA = "Merkaz"
+CITY = "Tel-Aviv"
+ADDRESS = "The best street 5"
+TEST_IMAGE_URL_01 = "../../static/images/daycare_image_test_01.jpeg"
+TEST_IMAGE_URL_02 = "../../static/images/daycare_image_test_02.jpeg"
```

Comment:
I don't think we need to use all of those constants. the content can be used explicitly on the fixture itself unless it repeats itself many times.

Reply:
I think that in that way, the constants are more readable and reusable for future pixtures implimantion

Reply:
This comment is relevant to the comment above am I right?  i.e. by using `pytest.mark.parametrize`, and it will be implemented in future PR

Reply:
Nope, because this is `conftest`, not `test_daycare`. here you don't have many tests, but different fixtures.

Reply:
You right, done :)